axelor-prestashop/src/main/java/com/axelor/apps/prestashop/imports/service/ImportAddressServiceImpl.java

```diff
@@ -122,6 +122,20 @@ public void importAddress(AppPrestashop appConfig, ZonedDateTime endDate, Writer
           partnerAddress.setIsDefaultAddr(Boolean.TRUE);
         }
         customer.addPartnerAddressListItem(partnerAddress);
+
+        String remoteAddressPhone = remoteAddress.getPhone();
+
+        if (customer.getFixedPhone() == null) {
+          customer.setFixedPhone(remoteAddressPhone);
+        } else if (remoteAddressPhone != null) {
+          String description = customer.getDescription();
+
+          if (description != null) {
+            customer.setDescription(description.concat(", " + remoteAddressPhone));
```

Comment:
Update description only if it doesn't contains phone number, else it will duplicate on next import.